CHANGELOG.md

```diff
@@ -22,7 +22,7 @@
 
 ## Improvements
 
--
+- Increase the information shown on VDM generation and parallelized the file creation process.
```

Comment:
One is past tense and one is present tense / imperative. I think we agreed to use imperative.
```suggestion
- Increase the information shown on VDM generation and parallelize the file creation process.
```